code/datums/status_effects/debuffs/debuffs.dm

```diff
@@ -958,5 +958,80 @@
 /datum/movespeed_modifier/careful_driving
 	multiplicative_slowdown = 3
 
+/datum/status_effect/midas_blight
+	id = "midas_blight"
+	alert_type = /atom/movable/screen/alert/status_effect/midas_blight
+	status_type = STATUS_EFFECT_REPLACE
+	tick_interval = 0.2 SECONDS
+	remove_on_fullheal = TRUE
+
+	/// The visual overlay state, helps tell both you and enemies how much gold is in your system
+	var/midas_state = "midas_1"
+	/// How fast the gold in a person's system scales.
+	var/goldscale = 30 // x2.8 - Gives ~ 15u for 1 second
+
+/datum/status_effect/midas_blight/on_creation(mob/living/new_owner, duration = 1)
+	// Duration is already input in SECONDS
+	src.duration = duration
+	RegisterSignal(new_owner, COMSIG_ATOM_UPDATE_OVERLAYS, PROC_REF(on_update_overlays))
+	return ..()
+
+/atom/movable/screen/alert/status_effect/midas_blight
+	name = "Midas Blight"
+	desc = "Your blood is being turned to gold, slowing your movements!"
+	icon_state = "midas_blight"
+
+/datum/status_effect/midas_blight/tick(seconds_between_ticks)
+	var/mob/living/carbon/human/victim = owner
+	victim.adjustOxyLoss(amount = 0.1, updating_health = TRUE, forced = TRUE) // Blood transmutation probably isn't very good for you
```

Comment:
does this actually outpace your natural oxyloss regen?

Reply:
good point, I have no clue. I'll look into that.